src/prompts/checkly-data.ts

```diff
@@ -2,24 +2,24 @@ import { ChecklyClient } from "../checkly/checklyclient";
 import { CheckResult } from "../checkly/models";
 
 export const LAST_30_DAYS = {
-  from: Date.now() - 30 * 24 * 60 * 60 * 1000,
-  to: Date.now(),
+  from: new Date(Date.now() - 30 * 24 * 60 * 60 * 1000),
+  to: new Date(),
 };
 
 export const LAST_24_HOURS = {
-  from: Date.now() - 24 * 60 * 60 * 1000,
-  to: Date.now(),
+  from: new Date(Date.now() - 24 * 60 * 60 * 1000),
```

Comment:
these are static, so they will not change after the application is initialised. I suggest to change them all to functions